src/remarx/app.py

```diff
@@ -213,7 +156,162 @@ def _(button, create_corpus, create_temp_input, input_file, mo, output_csv):
+        This process can be run multiple times for different files (currently one file at a time).
+
+        **1. Select Input Text**
+
+        Upload and select an input file (`{"`, `".join(FileInput.supported_types())}`) for sentence corpus creation.
+        Currently, only a single file may be selected.
+        """
+        ).style(width="100%"),
+
+        input_selection_ui,
+
+        mo.md(
+            """
+        **2. Select Output Location**
+
+        Select the folder where the resulting sentence corpus file should be saved.
+        The output CSV file will be named based on the input file.
+
+        *To select a folder, click the file icon to the left of the folder's name.
+        A checkmark will appear when a selection is made.
+        Clicking anywhere else within the folder's row will cause the browser to navigate to this folder and subsequently display any folders *within* this folder.*
+        """
+        ).style(width="100%"),
+
+        output_selection_ui,
+
+        mo.md(
+            """
+        **3. Build Sentence Corpus**
+
+        Click the "Build Corpus" to run `remarx`.
+        The sentence corpus for the input text will be saved as a CSV in the selected save location.
+        This output file will have the same filename (but different file extension) as the selected input file.
+        """
+        ).style(width="100%"),
+
+        build_corpus_ui,
+        corpus_status_ui,
+    ])
+
+    return (sentence_corpus_creation_content,)
+
+
+@app.cell
+def _(mo, pathlib):
+    # Create file browsers for quotation detection (CSV files only)
+    original_csv_browser = mo.ui.file_browser(
+        selection_mode="file",
+        multiple=True,
+        initial_path=pathlib.Path.home(),
+        filetypes=[".csv"],
+    )
+
+    reuse_csv_browser = mo.ui.file_browser(
+        selection_mode="file",
+        multiple=True,
+        initial_path=pathlib.Path.home(),
+        filetypes=[".csv"],
+    )
+
+    return (original_csv_browser, reuse_csv_browser)
+
+
+@app.cell
+def _(mo, original_csv_browser, reuse_csv_browser):
+    # Process file selections for quotation detection
+    original_csvs = original_csv_browser.value if original_csv_browser.value else []
+    reuse_csvs = reuse_csv_browser.value if reuse_csv_browser.value else []
+
+    original_count = len(original_csvs)
+    reuse_count = len(reuse_csvs)
+
+    original_msg = "" if original_count > 0 else "No original text files selected"
+    reuse_msg = "" if reuse_count > 0 else "No reuse text files selected"
+
+    original_callout_type = "success" if original_count > 0 else "warn"
+    reuse_callout_type = "success" if reuse_count > 0 else "info" # use a different color
+
+    # Create side-by-side file browser interface
+    quotation_file_selection_ui = mo.hstack([
+        mo.callout(
+            mo.vstack([
+                mo.md("# 🗂").center(),
+                mo.md("**Original Text CSV Files**").center(),
```

Comment:
We should be consistent about using the name we are calling this input.
```suggestion
                mo.md("**Select Original Sentence Corpora (CSVs)**").center(),
```